### DISCUSSION TITLE
worldguard regions

### DISCUSSION THREAD

**Discussion by JonatanLaMerce** [TownyAdvanced/Towny#5228] [Q&A]
hello, is it possible that it is not possible to create a worldguard region within a town region?

I mean, I have a plugin that creates regions in worldguard and I would like to know if it is possible to not let create these regions within a towny region, or deny within a towny region the command that creates this region. 

Thank you very much.

**Comment by LlmDl** [comment]
https://github.com/TownyAdvanced/WorldGuard-Towny might interest you. With it you can stop a town being made in a WG region.

Towny does not have the ability to stop a WG region being made in a town.
